src/main/resources/swagger/agora.yaml

```diff
@@ -255,10 +260,7 @@ paths:
         - Methods
       responses:
         '200':
-          description: |
-            An integer showing how many methods were deleted.
-          schema:
-            type: number
+          description: An integer showing how many methods were deleted.
```

Comment:
this works in our version of swagger-ui ... but it's also a syntax error for swagger parsers. Grr. I'm thinking on what to do here ... bottom line is I prefer working over correct, but it would be nice to have both.

Update:
if you change this to be:
```
        '200':
          description: An integer showing how many methods were deleted.
          schema:
            $ref: '#/definitions/Number'
```
and then add, inside the definitions block,
```
  Number:
    type: number
    example: 1
```
the code is syntactically correct, and the swagger-ui displays something useful.

Reply:
Ha, it's like autoboxing but for Swagger. Great idea!